services/__tests__/time-entries-api.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { getTimeEntries } from "../time-entries-api";
+import { NotFoundError } from "../errors";
+
+const mockedTimeEntries = [
```

Comment:
You can put these in a __mocks__ folder. Link in slack fe-crashcourse-channel to jest documentation.